### PR TITLE
#84: queries.filter should be per-query (closes #84)

### LINKED ISSUE #84: queries.filter should be per-query
currently can cause duplicate query declarations to be filtered out even if only one `filter` condition is `false`

### PR DESCRIPTION
**Pull request by giogonzo**
Issue #84

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -77,19 +77,20 @@ export default class App {
...
-    }).filter(v => v).reduce((ac, q) => ({
-      ...ac,
-      ...q
-    }), {});
+        log(`${route.handler.name} queries: %o %o %o %o`, route, route.handler, route.handler.prototype, route.handler.queries ? route.handler.queries : 'no qs');
+        return route.handler.queries;
+      })
+      .filter(v => v)
+      .reduce((ac, qs) => ({  ...ac, ...(
+        Object.keys(qs)
```
Comment (francescogior) [inline src/App.js:85]: what about queries declared as strings?

Comment (giogonzo) [inline src/App.js:85]: at this point they all have this uniform shape here. See https://github.com/buildo/revenge/blob/revenge-84-queriesfilter_should_be_perquery/src/decorators/queries.js

Comment (francescogior) [inline src/App.js:85]: :+1:

**Comment by giogonzo** [comment]
adding no unit test here as this will hopefully be rewritten soon and change api

**Pull request merged**